outputs.tf

```diff
@@ -15,6 +15,11 @@
  */
 
 output "name" {
-  value       = google_cloud_scheduler_job.job.name
+  value       = google_cloud_scheduler_job.job.*.name
```

Comment:
This output is a list: I think to make it work it should be `null` or a `string`, thoughts ?

Reply:
Yes, need to think how to solve this. Though the second approach at https://github.com/terraform-google-modules/terraform-google-scheduled-function/pull/27 does not have such problem and does not require using `pubsub` module's `enable` or `create_topic` variables. We wouldn't need making any changes in the terraform-google-scheduled-function module using the existing functionality and the same pubsub topic as described at https://github.com/terraform-google-modules/terraform-google-scheduled-function/issues/15#issuecomment-564426393